app/controllers/api/v2/registration_controller.rb

```diff
@@ -23,13 +23,20 @@ class RegistrationController < V2::BaseController
       param :setup_remote_execution, :bool, desc: N_("Set 'host_registration_remote_execution' parameter for the host. If it is set to true, SSH keys will be installed on the host")
       param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter, example: `pkg1 pkg2`")
       param :update_packages, :bool, desc: N_("Update all packages on the host")
+      param :repo, String, desc: N_("Repository URL / details, for example for Debian OS family: 'deb http://deb.example.com/ buster 1.0', for Red Hat and SUSE OS family: 'http://yum.theforeman.org/client/latest/el8/x86_64/'"), deprecated: true
```

Comment:
When I run the hammer command
```
bundle exec bin/hammer host-registration generate-command --repo "https://repo.com"
```

I don't see any deprecation warning in Foreman / Hammer log.

@nadjaheitmann does it work for you?

Reply:
I can see it in the console output of my Forklift so I assume it would appear in production log. Not sure how helpful that is, though.

Reply:
Well, I can see it if I run the command, not for the help output.